### ISSUE TITLE
Calling command method with apply method crashes entire z-way server

### ISSUE THREAD

**Issue by shreking-around** [Z-Wave-Me/home-automation#467]
When I try to call command method, for example Set and I want to provide its arguments via array it should be possible to use apply() method. Instead the entire z-way server crashes - the service still runs but the whole JSON API is inaccessible. 

For example:

http://192.168.1.21:8083/ZWaveAPI/Run/zway.devices[13].instances[0].commandClasses[67]["Set"].apply(null,[1,25])

Desired outcome: Set should be executed with provided arguments
Actual outcome: Z-way server crashes

I found nothing in the logs, however I found this entry in syslog: 
`Feb 28 11:11:18 raspberrypi kernel: [   61.637032] Unhandled prefetch abort: breakpoint debug exception (0x002) at 0x76de8950`

Kernel: Linux raspberrypi 4.9.66-v7+ #1059 SMP Fri Dec 1 15:38:42 GMT 2017 armv7l GNU/Linux
OS: Raspbian GNU/Linux 8 
Firmware Version: 2.3.6

**Comment by PoltoS** [comment]
What do you expect from such a call? You are calling a mathod changing _this_ to _null_! Z-Way can not access the context and crashes. It is definitelly bad that we don't check for _null_ in the C++ code, but we never expected the code to survive method borrowing - what shall we do whn you call the method on _null_?

Nobody before you did that and hope nobody will ever do in future ;) I would call it not a bug, but a misuse.

**Comment by shreking-around** [comment]
The null argument is just a illustration, I have tried different contexts but with no luck. I am trying to call it from ZAutomationAPIProvider.js but will no luck. Sometimes I get this error, for example when I call it with this. 
```
[2018-03-02 10:20:11.558] [E] [HTTP] Callback execution error: Error: Binding was stopped
    at Error (native)
    at ZAutomationAPIWebRequest._.extend.executeCommand (automation/ZAutomationAPIProvider.js:447:80)
    at ZAutomationAPIWebRequest.ZAutomationWebRequest.handleRequest (automation/WebserverRequestRouter.js:259:35)
    at automation/WebserverRequestRouter.js:32:29
    at WebServer.document_root (automation/Webserver.js:35:11)
```
Well it is maybe unexpected bug but I would call it a security issue - it is very easy to crash the z-way server from web browser this way. Would you please advise which context I should provide the Set method? Thank you for your help.

**Comment by PoltoS** [comment]
Agree, we will keep the issue open to fix it. Hopefully only admin can exploit it, so don't worry about security.

The correct context is the CommanClass itself - like in other OOP languages.

Try this:
`http://192.168.1.21:8083/ZWaveAPI/Run/zway.devices[13].instances[0].commandClasses[67]["Set"].apply(zway.devices[13].instances[0].commandClasses[67],[1,25])`

Why do you do it, btw? Why not just `zway.devices[13].instances[0].commandClasses[67].Set(1, 25)` ?
